src/views/meetings/midweek/S140/app_normal/index.types.ts

```diff
@@ -9,6 +9,7 @@ export type S140PartTimeType = {
   time: string;
   color: string;
   backgroundColor: string;
+  isClosingSong: boolean;
```

Comment:
@mrtavor: for this case it would be great if we make the `isClosingSong` prop as optional: `isClosingSong?: boolean`. Then you can remove all the `isClosingSong={false}`, and only keep it for the one instance that needs it. Thanks.